usaspending_api/etl/es_award_template.json

```diff
@@ -0,0 +1,442 @@
+          "fields": {
+            "keyword": {
+              "type": "keyword"
+            }
+          }
+        },
+        "pop_country_code": {
+          "type": "keyword"
+        },
+        "pop_state_code": {
+          "type": "keyword"
+        },
+        "pop_county_code": {
+          "type": "keyword"
+        },
+        "pop_county_name": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword"
+            }
+          }
+        },
+        "pop_zip5": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword"
+            }
+          }
+        },
+        "pop_congressional_code": {
+          "type": "keyword"
+        },
+        "pop_city_name": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword"
+            }
+          }
+        },
+        "cfda_number": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword"
+            }
+          }
+        },
+        "sai_number": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword"
+            }
+          }
+        },
+        "type_of_contract_pricing": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword"
+            }
+          }
+        },
+        "type_set_aside": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword"
+            }
+          }
+        },
+        "extent_competed": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword"
+            }
+          }
+        },
+        "product_or_service_code": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword"
+            }
+          }
+        },
+        "product_or_service_description": {
+          "type": "text"
+        },
+        "naics_code": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword"
+            }
+          }
+        },
+        "naics_description": {
+          "type": "text"
+        },
+        "treasury_accounts": {
+          "type": "nested",
+          "properties": {
+            "aid": {
+              "type": "integer"
+            },
+            "ata": {
+              "type": "integer"
+            },
+            "main": {
+              "type": "integer"
+            },
+            "sub": {
+              "type": "integer"
+            },
+            "bpoa": {
+              "type": "integer"
+            },
+            "epoa": {
+              "type": "integer"
+            },
+            "a": {
+              "type": "text"
+            }
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
nice to include a trailing blank line for all text files (python, json, sql, ...)